hack/update-codegen.sh

```diff
@@ -71,4 +73,19 @@ ${clientgen} --clientset-name=federation_internalclientset --clientset-path=k8s.
 ${clientgen} --clientset-name=federation_release_1_5 --clientset-path=k8s.io/kubernetes/federation/client/clientset_generated --input="../../federation/apis/federation/v1beta1","api/v1","extensions/v1beta1" --included-types-overrides="api/v1/Service,api/v1/Namespace,extensions/v1beta1/ReplicaSet,api/v1/Secret,extensions/v1beta1/Ingress,extensions/v1beta1/Deployment,extensions/v1beta1/DaemonSet,api/v1/ConfigMap,api/v1/Event"   "$@"
 ${setgen} "$@"
 
+LISTERGEN_APIS=(
+pkg/api
+pkg/api/v1
+$(
+  pushd ${KUBE_ROOT} > /dev/null
```

Comment:
Yes, I need it because otherwise I have to do `find ${KUBE_ROOT}/pkg/apis` which returns the full absolute path, which I would then have to strip off (I need the importable go package name, not the file's location on disk).